uwblueprint/marillac-place
# API Call to Add Resident

## Notion ticket link
[Participant Page - Adding Residents](https://www.notion.so/uwblueprintexecs/F24-Task-Board-10310f3fb1dc8078bdfbee0b90179dc3?p=12910f3fb1dc80f498fad0d8a3a92b1d&pm=s)



## Implementation description
* Called GraphQL mutation to add resident using constant in APICalls
* Adds inputted data except for some fields (such as birth date)
* Added error message


## Steps to test
1. Click Add Resident and input all necessary fields
2. Click Add Resident and the page should reload with the new resident added



## What should reviewers focus on?
* Inspect element and there shouldn't be any errors in the Console


## Checklist
